content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-05-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+# TITLE + DATE
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of May 2020.
+
+We are always happy about contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. 
+The drafting process for each newsletter [is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+Furthermore, we started drafting each new version in the [XSF Github repository](https://github.com/xsf/xmpp.org/pulls) - feel free to add information by yourself!
+
+[Subscribe](https://tinyletter.com/xmpp) to receive the next edition in your inbox!
+
+### Newsletter translations
+
+The translations of the XMPP Newsletter will be released here:
+
+- [German](https://www.jabber.de/)
+- [French](https://linuxfr.org/tags/xmpp/public)
+- [Spanish](https://www.jabber.de/)
+
+
+## XMPP Standards Foundation
+
+## Articles
+
+_Everyone participating, please just don't drop links rather try to make a short and decent 
+formulated description what is the main outcome of your source. This will help us a lot! 
+You don't need to write formal, be passionately why this motivates you!_
+
+## Software releases
+
+_Everyone participating, please just don't drop links rather try to make a short and decent 
+formulated description what is the main outcome of your source. This will help us a lot! 
+You don't need to write formal, be passionately why this motivates you!_
+
+### Servers
+
+### Clients and applications
+
+## Services
+
+_Everyone participating, please just don't drop links rather try to make a short and decent 
+formulated description what is the main outcome of your source. This will help us a lot! 
+You don't need to write formal, be passionately why this motivates you!_
+
+## Libraries
+
+
+# Extensions and specifications
+
+### Updated
+
+### Proposed
+
+### Miscellaneous
+
+**Experimental**
+
+**Draft**
+
+....
+
+
+
+## Thanks all!
+
+This XMPP Newsletter is produced collaboratively by the community.
+
+Thanks to [CONTRIBUTORS] for their help in creating it!
+
+Please share the news on "social networks":
+
+* Twitter: https://twitter.com/xmpp
+* Mastodon: https://fosstodon.org/@xmpp/
+* LinkedIn: https://www.linkedin.com/company/xmpp-standards-foundation/
+* Facebook: https://www.facebook.com/jabber/
+* Reddit: https://www.reddit.com/r/xmpp/
+
```

Comment:
I suggest that we remove all of the raw URLs from the newsletter and use the [caption](link) syntax when it is possible, including for the above list of social networks accounts:
`Twitter: https://twitter.com/xmpp` => `[Twitter](https://twitter.com/xmpp)`